Backend/app/main.py

```
@@ -102,6 +115,330 @@ def get_session():
+        }
+    except HTTPException:
+        raise
+    except Exception as e:
+        raise HTTPException(
+            status_code=500,
+            detail=f"Failed to refresh token: {str(e)}"
+        )
+
+@router_login.post("/admin/refresh-expired-tokens", tags=["admin"])
+def admin_refresh_all_expired_tokens(
+    session: Session = Depends(lambda: Session(engine)),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)